src/documentCreator/test/fixture/config.json

```diff
@@ -0,0 +1,245 @@
+{
+  "documentMeta": {
+    "name": "Maersk Bill of Lading",
+    "$template": {
+      "name": "BILL_OF_LADING",
+      "type": "EMBEDDED_RENDERER",
+      "url": "https://demo-cnm.openattestation.com"
+      },
+    "issuers": [
+        {
+        "name": "DEMO STORE",
+        "tokenRegistry": "0xdA8DBd2Aaffc995F11314c0040716E791de5aEd2",
+        "identityProof": {
+          "type": "DNS-TXT",
+          "location": "tradetrust.io"
+          }
+        }
+      ]
+    },
+  "wallet": {
+    "address":"67b5bc373caa527cee654d6a0f629ba1e84fad02",
+    "id":"c829be9f-2e80-4425-a3b1-0170d5cb6a6a",
+    "version":3,
+    "Crypto":{
+      "cipher":"aes-128-ctr",
+      "cipherparams":{ 
+        "iv":"815f141a79f7b5974cb7748d6d374e9e"
+        },
+        "ciphertext":"b430379e7c652e8ca2138cc1a04a6dd6b9a245b6ffe48dddc89a880b8ecc450f",
+        "kdf":"scrypt",
+        "kdfparams":{
+          "salt":"046b5efd97aae85365c55b9ef8b663dc931ec13e9471ca784f31fe911d733fea",
+          "n":131072,
+          "dklen":32,
+          "p":1,
+          "r":8},
+          "mac":"d5645e5efe60bc5c84f6433f5052d365d8303910c8c745a43f23e55deac49832"
+          }
+        },
+  "formSchema": [{
+  "$id": "xyz",
```

Comment:
Are you sure this json has been formatted? This looks like the indentation is off.